src/store/actions.js

```diff
@@ -165,9 +176,27 @@ export default {
    * Payload same as executeTurn.
    */
   groupStacks (context, payload) {
+   // let stacks = context.state.stacks
+    let stackValue= []
+    for (let stack of payload.target){
+      stackValue.push(stack.getScore)
+      
+      
+    }
     context.commit('removeStacks', {stacks: payload.target})
     context.commit('newStack', payload)
     context.commit('removeFromHand', payload)
+    log.warn(payload.player.name, payload.card.type, payload.card.value,stackValue)
+    
+   //if (payload.card.type === "GROUP") {
```

Comment:
If this code is not going to be used it should be removed along with the blank lines.